Nutraceutical Additions

RLordan opened the pull request:
Hey folks, some additions to vitamin D and n-3 fatty acid research. Will re-evaluate the other at the weekend. If anyone would be free to review I would be very grateful! :)

<!-- Hi there! Please use the template below as a guide for opening a pull request.
This template is designed to help the pull request process go smoother.
If anything doesn't make sense to you, please open the pull request anyway and leave entries blank as needed.
Thank you for your contribution!! -->

### Description of the proposed additions or changes
<!-- If the additions or changes aren't self explanatory feel free to leave a description or notes here. -->


### Related issues
<!-- GitHub will link issues to pull requests if you list the related issue numbers like "#2, #17". -->


### Suggested reviewers (optional)
<!-- If there are particular people you think should review this pull request, please list their GitHub handle like "@rando2".-->


### Checklist
<!-- Please mark the following tasks as complete by putting an "x" in the box like "[x]".
If a particular item is not applicable to your pull request, please delete the item. -->
- [ ] Text is formatted so that each sentence is on its own line.
<!-- see why this is recommended here:https://github.com/greenelab/covid19-review/blob/627a5f45a2d1fcb2150d1466df63bb848f22a3d4/USAGE.md#manubot-markdown -->
- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

RLordan commented:
@rando2 @cgreene feel free to edit :)

rando2 commented on content/20.treatments.md:
```diff
@@ -472,13 +481,24 @@ Due to its potential immunomodulating properties, vitamin D supplementation may
-A recent preprint postulated that an individual’s vitamin D status may significantly affect one’s risk of developing COVID-19.
-This hypothesis was derived from the fact that the current pandemic occurred in winter in Wuhan China when 25-hydroxyvitamin D concentrations are at their lowest due to lack of sunlight, whereas in the Southern Hemisphere, where it was nearing the end of the summer, the number of cases was low at that time coinciding with higher 25-hydroxyvitamin D concentrations [@doi:10.20944/preprints202003.02 ...
+A recent review postulated that an individual’s vitamin D status may significantly affect one’s risk of developing COVID-19.
+This hypothesis was derived from the fact that the current pandemic occurred in winter in Wuhan China when 25-hydroxyvitamin D concentrations are at their lowest due to lack of sunlight, whereas in the Southern Hemisphere, where it was nearing the end of the summer, the number of cases was low at that time coinciding with higher 25-hydroxyvitamin D concentrations [@doi:10.3390/nu12040988].
```
```suggestion
This hypothesis was derived from the fact that the current pandemic emerged in winter in Wuhan China when 25-hydroxyvitamin D concentrations are at their lowest due to a lack of sunlight, whereas in the Southern Hemisphere, where it was nearing the end of the summer and higher 25-hydroxyvitamin D concentrations would be higher, the number of cases was low [@doi:10.3390/nu12040988].
```

rando2 commented on content/20.treatments.md:
```diff
@@ -472,13 +481,24 @@ Due to its potential immunomodulating properties, vitamin D supplementation may
 The authors suggest that people at risk of developing COVID-19 should increase their vitamin D3 intake to reach 25-hydroxyvitamin D plasma concentrations above 40–60 ng/ml.
 The authors also suggest supplementation of vitamin D to treat infected patients and to prevent infection in hospital staff.
-While vitamin D is relatively inexpensive and safe to consume, caution is warranted when interpreting this review as it has yet to be determined whether vitamin D levels effect COVID-19 specifically.
+While vitamin D is relatively inexpensive and safe to consume, caution is warranted when interpreting this review as it has yet to be determined whether vitamin D levels effect COVID-19 specifically.  
```
```suggestion
While vitamin D is relatively inexpensive and safe to consume, caution is warranted when interpreting this review as it has yet to be determined whether vitamin D levels affect COVID-19 outside of this geographic/climatic correlation.  
```

RLordan replied:
effect/affect deadly sin by me!!!

rando2 commented on content/20.treatments.md:
```diff
@@ -472,13 +481,24 @@ Due to its potential immunomodulating properties, vitamin D supplementation may
 The authors also suggest supplementation of vitamin D to treat infected patients and to prevent infection in hospital staff.
-While vitamin D is relatively inexpensive and safe to consume, caution is warranted when interpreting this review as it has yet to be determined whether vitamin D levels effect COVID-19 specifically.
+While vitamin D is relatively inexpensive and safe to consume, caution is warranted when interpreting this review as it has yet to be determined whether vitamin D levels effect COVID-19 specifically.  
 Likewise, it is assumed that COVID-19 may be seasonal, but there are multiple other factors at play that can affect vitamin D levels that need to be considered.
```
```suggestion
Likewise, though it is assumed that COVID-19 may be seasonal, multiple other factors that can affect vitamin D levels should also be considered.
```

rando2 commented on content/20.treatments.md:
```diff
@@ -472,13 +481,24 @@ Due to its potential immunomodulating properties, vitamin D supplementation may
 Likewise, it is assumed that COVID-19 may be seasonal, but there are multiple other factors at play that can affect vitamin D levels that need to be considered.
-These factors include an individual’s nutritional status, their age, their occupation, skin pigmentation, potential comorbidities, and the variation of exposure to sunlight due to latitude amongst others [@doi:10.3390/nu7075111].   
+These factors include an individual’s nutritional status, their age, their occupation, skin pigmentation, potential comorbidities, and the variation of exposure to sunlight due to latitude amongst others [@doi:10.3390/nu7075111]. 
+As the pandemic evolves, research has started to interrogate many of the potential links identified in the Grant et al. review [@doi:10.3390/nu12040988] between vitamin D and COVID-19 in terms of its prophylactic and therapeutic value. 
```
```suggestion
As the pandemic evolves, further research has investigated some of the potential links identified in the Grant et al. review [@doi:10.3390/nu12040988] between vitamin D and COVID-19 and sought to shed light on whether there is any  prophylactic and/or therapeutic relationship. 
```

rando2 commented on content/20.treatments.md:
```diff
@@ -472,13 +481,24 @@ Due to its potential immunomodulating properties, vitamin D supplementation may
-These factors include an individual’s nutritional status, their age, their occupation, skin pigmentation, potential comorbidities, and the variation of exposure to sunlight due to latitude amongst others [@doi:10.3390/nu7075111].   
+These factors include an individual’s nutritional status, their age, their occupation, skin pigmentation, potential comorbidities, and the variation of exposure to sunlight due to latitude amongst others [@doi:10.3390/nu7075111]. 
+As the pandemic evolves, research has started to interrogate many of the potential links identified in the Grant et al. review [@doi:10.3390/nu12040988] between vitamin D and COVID-19 in terms of its prophylactic and therapeutic value. 
+A study in Switzerland demonstrated that 27 SARS-CoV-2 positive patients exhibited 25-hydroxyvitamin D plasma concentrations that were significantly lower (11.1 ng/ml) than that of SARS-CoV-2 negative patients (24.6 ng/ml; p = 0.004), which was also confirmed when stratifying for patients greater than 70 years old [@doi:10.3390/nu12051359]. 
```
```suggestion
A study in Switzerland demonstrated that 27 SARS-CoV-2 positive patients exhibited 25-hydroxyvitamin D plasma concentrations that were significantly lower (11.1 ng/ml) than those of SARS-CoV-2 negative patients (24.6 ng/ml; p = 0.004), an association that held when stratifying for patients greater than 70 years old [@doi:10.3390/nu12051359]. 
```

rando2 commented on content/20.treatments.md:
```diff
@@ -472,13 +481,24 @@ Due to its potential immunomodulating properties, vitamin D supplementation may
+These factors include an individual’s nutritional status, their age, their occupation, skin pigmentation, potential comorbidities, and the variation of exposure to sunlight due to latitude amongst others [@doi:10.3390/nu7075111]. 
+As the pandemic evolves, research has started to interrogate many of the potential links identified in the Grant et al. review [@doi:10.3390/nu12040988] between vitamin D and COVID-19 in terms of its prophylactic and therapeutic value. 
+A study in Switzerland demonstrated that 27 SARS-CoV-2 positive patients exhibited 25-hydroxyvitamin D plasma concentrations that were significantly lower (11.1 ng/ml) than that of SARS-CoV-2 negative patients (24.6 ng/ml; p = 0.004), which was also confirmed when stratifying for patients greater than 70 years old [@doi:10.3390/nu12051359]. 
+These findings seem to be supported by another observational study of 186 SARS-CoV-2 positive patients exhibiting symptoms of pneumonia in Belgium, whereby 25-hydroxyvitamin D plasma concentrations were measured and a CT scan of the lungs was obtained upon hospitalisation [@doi:10.1101/2020.05.01.20079376]. 
```
```suggestion
These findings seem to be supported by a Belgian observational study of 186 SARS-CoV-2 positive patients exhibiting symptoms of pneumonia, where 25-hydroxyvitamin D plasma concentrations were measured and a CT scan of the lungs was obtained upon hospitalization [@doi:10.1101/2020.05.01.20079376]. 
```

rando2 commented on content/20.treatments.md:
```diff
@@ -472,13 +481,24 @@ Due to its potential immunomodulating properties, vitamin D supplementation may
+As the pandemic evolves, research has started to interrogate many of the potential links identified in the Grant et al. review [@doi:10.3390/nu12040988] between vitamin D and COVID-19 in terms of its prophylactic and therapeutic value. 
+A study in Switzerland demonstrated that 27 SARS-CoV-2 positive patients exhibited 25-hydroxyvitamin D plasma concentrations that were significantly lower (11.1 ng/ml) than that of SARS-CoV-2 negative patients (24.6 ng/ml; p = 0.004), which was also confirmed when stratifying for patients greater than 70 years old [@doi:10.3390/nu12051359]. 
+These findings seem to be supported by another observational study of 186 SARS-CoV-2 positive patients exhibiting symptoms of pneumonia in Belgium, whereby 25-hydroxyvitamin D plasma concentrations were measured and a CT scan of the lungs was obtained upon hospitalisation [@doi:10.1101/2020.05.01.20079376]. 
+A significant difference of 25-hydroxyvitamin D levels was observed between the SARS-CoV-2 patients and 2,717 season-matched diseased controls.
```
```suggestion
A significant difference in 25-hydroxyvitamin D levels was observed between the SARS-CoV-2 patients and 2,717 season-matched diseased controls.
```

rando2 commented on content/20.treatments.md:
```diff
@@ -472,13 +481,24 @@ Due to its potential immunomodulating properties, vitamin D supplementation may
+These findings seem to be supported by another observational study of 186 SARS-CoV-2 positive patients exhibiting symptoms of pneumonia in Belgium, whereby 25-hydroxyvitamin D plasma concentrations were measured and a CT scan of the lungs was obtained upon hospitalisation [@doi:10.1101/2020.05.01.20079376]. 
+A significant difference of 25-hydroxyvitamin D levels was observed between the SARS-CoV-2 patients and 2,717 season-matched diseased controls.
+Both female and male patients possessed lower median 25-hydroxyvitamin D concentrations than the control group (18.6 ng/ml versus 21.5 ng/ml; p = 0.0016) and a higher rate of vitamin D deficiency (58.6% versus 42.5%). 
+Evidence of sexual dimorphism was reported as female patients had equivalent levels of 23-hydroxyvitamin D to the control group, whereas male patients were particularly deficient in 25-hydroxyvitamin D versus the controls (67% versus 49%; p = 0.0006). 
```
```suggestion
Evidence of sexual dimorphism was apparent, as female patients had equivalent levels of 23-hydroxyvitamin D to the control group, whereas male patients were deficient in 25-hydroxyvitamin D relative to male controls (67% versus 49%; p = 0.0006). 
```

rando2 commented on content/20.treatments.md:
```diff
@@ -472,13 +481,24 @@ Due to its potential immunomodulating properties, vitamin D supplementation may
+Despite these two observational studies potentially linking vitamin D with COVID-19, an examination of the UK Biobank did not support this thesis [@doi:10.1016/j.dsx.2020.04.050]. 
+This analysis examined 25-hydroxyvitamin D concentrations in 348,598 UK Biobank participants, of which 449 were SARS-CoV-2 positive. 
+There is significant interest in the link between vitamin D and COVID-19, hence the multitude of studies published in the literature of varying quality and the clinical trials underway.
+One trial is currently investigating the utility of vitamin D as an immune modulating agent by monitoring  if there is an improvement of health status in non-severe symptomatic patients infected with COVID-19 as well as determining if vitamin D prevents patient deterioration [@url: https://clinicaltrials.gov/ct2/show/NCT04334005].
```
```suggestion
One trial is currently investigating the utility of vitamin D as an immune-modulating agent by monitoring whether administration of vitamin D precipitates an improvement of health status in non-severe symptomatic patients infected with COVID-19 or whether vitamin D prevents patient deterioration [@url: https://clinicaltrials.gov/ct2/show/NCT04334005].
```

rando2 commented on content/20.treatments.md:
```diff
@@ -491,6 +511,7 @@ This measure is likely to limit people’s exposure to sunlight and thus reduce
 Despite all the potential benefits of nutraceutical and dietary supplement interventions presented, currently there is a paucity of clinical evidence to support their use for the prevention or mitigation of COVID-19 infections.
 Nevertheless, optimal nutritional status will undoubtedly prime an individual’s immune system to protect against the effects of acute respiratory viral infections by supporting normal maintenance of the immune system [@url:https://www.preprints.org/manuscript/202003.0199/v1].
 Overall, supplementation of vitamin C, vitamin D, and zinc may be an effective method of ensuring their adequate intake to maintain optimal immune function, which may also convey beneficial effects against viral infections due to their immunomodulatory effects.
+As a result, the administration of zinc, vitamin C, and vitamin D in conjunction with hydroxychloroquine are being investigated for their prophylactic effects in healthcare workers and their first degree relatives in Turkey [@url:https://clinicaltrials.gov/ct2/show/NCT04326725].
```
```suggestion
As a result, the administration of zinc, vitamin C, and vitamin D in conjunction with hydroxychloroquine are being investigated for their prophylactic effects in healthcare workers and their first-degree relatives in Turkey [@url:https://clinicaltrials.gov/ct2/show/NCT04326725].
```

agitter commented:
Let's try to merge the major update in #308 before finishing this pull request

RLordan commented on content/20.treatments.md:
```diff
@@ -390,8 +390,17 @@ Another potential mechanism by which n-3 PUFA could exert beneficial effects aga
+It is hypothesised that SPM may aid in the resolution of the cytokine storm and pulmonary inflammation associated with COVID-19 [@doi:10.1007/s10555-020-09889-4].
 Moreover, not all studies are in agreement that n-3 PUFA are effective against infections [@doi:10.3945/jn.109.108027].
-Indeed, the effectiveness of n-3 PUFA against infections is dependent on the dosage, timing, and the specific pathogens responsible [@doi:10.1016/j.jinf.2016.10.001].  
+The effectiveness of n-3 PUFA against infections is dependent on the dosage, timing, and the specific pathogens responsible [@doi:10.1016/j.jinf.2016.10.001].
```
The literature in relation to n-3 fatty acids is not there yet for infection specifically. Hence my phrasing not being particularly definitive I guess. However, they are valued for their anti-inflammatory properties. In any case, evidence shows that the dosage and timing  are important but there is no clear consensus on that for several reasons. I will review again.

RLordan commented:
> These are just superficial comments! I will do the skim-the-literature-and-make-sure-the-takeaways-checkout read through in the morning and let you know if I see anything we should discuss further. Thank you for updating!

Thanks @rando2 , much appreciated as always!

RLordan commented on content/20.treatments.md:
```diff
@@ -390,8 +390,17 @@ Another potential mechanism by which n-3 PUFA could exert beneficial effects aga
+
+One pathological aspect of COVID-19 infection that was late to be recognised  was the increased risk of thrombotic complications [@doi:10.1016/j.thromres.2020.04.013]. 
+Considering, there is significant evidence that n-3 fatty acids and other fish oil-derived lipids possess antithrombotic properties and anti-inflammatory properties [@doi:10.1016/j.prostaglandins.2018.09.005;@doi:10.1016/j.blre.2020.100694] they may have therapeutic value against the prothrombotic complications of COVID-19. 
+This may be particularly important since patients receiving pharmaceutical antiplatelet therapies may become infected.
```
I have made this section clearer. That was my bad, see if it makes more sense now :)

matfax commented on content/20.treatments.md:
```diff
@@ -472,13 +482,24 @@ Due to its potential immunomodulating properties, vitamin D supplementation may
 
-A recent preprint postulated that an individual’s vitamin D status may significantly affect one’s risk of developing COVID-19.
-This hypothesis was derived from the fact that the current pandemic occurred in winter in Wuhan China when 25-hydroxyvitamin D concentrations are at their lowest due to lack of sunlight, whereas in the Southern Hemisphere, where it was nearing the end of the summer, the number of cases was low at that time coinciding with higher 25-hydroxyvitamin D concentrations [@doi:10.20944/preprints202003.02 ...
+A recent review postulated that an individual’s vitamin D status may significantly affect their risk of developing COVID-19.
```
I would stick to the Calcifediol/Calcitriol or 25-OH/1,25-OH terminology and introduce them here because they are short and comprehensible. Where is the preprint reference?

matfax commented on content/20.treatments.md:
```diff
@@ -472,13 +482,24 @@ Due to its potential immunomodulating properties, vitamin D supplementation may
-Likewise, it is assumed that COVID-19 may be seasonal, but there are multiple other factors at play that can affect vitamin D levels that need to be considered.
-These factors include an individual’s nutritional status, their age, their occupation, skin pigmentation, potential comorbidities, and the variation of exposure to sunlight due to latitude amongst others [@doi:10.3390/nu7075111].   
+While vitamin D is relatively inexpensive and safe to consume, caution is warranted when interpreting this review as it has yet to be determined whether vitamin D levels affect COVID-19 outside of this geographic/climatic correlation.  
+Likewise, though it is assumed that COVID-19 may be seasonal, multiple other factors that can affect vitamin D levels should also be considered.
```
From my understanding, a seasonal disease is something different than a disease with seasonal severity. What exactly is the assumption of this review?

RLordan replied:
This line refers to seasonal incidence (more infection around January etc.), in line with the hypothesis that vitamin D is a potential factor.
From my understanding, there has been little discussion on potential seasonal severity (the human impact of the virus e.g. increased hospitalisation levels etc.); Although, viral mutation patterns are being continually monitored.

matfax commented on content/20.treatments.md:
```diff
@@ -472,13 +482,24 @@ Due to its potential immunomodulating properties, vitamin D supplementation may
+While vitamin D is relatively inexpensive and safe to consume, caution is warranted when interpreting this review as it has yet to be determined whether vitamin D levels affect COVID-19 outside of this geographic/climatic correlation.  
+Likewise, though it is assumed that COVID-19 may be seasonal, multiple other factors that can affect vitamin D levels should also be considered.
+These factors include an individual’s nutritional status, their age, their occupation, skin pigmentation, potential comorbidities, and the variation of exposure to sunlight due to latitude amongst others [@doi:10.3390/nu7075111]. 
+As the pandemic evolves, further research has investigated some of the potential links identified in the Grant et al. review [@doi:10.3390/nu12040988] between vitamin D and COVID-19 and sought to shed light on whether there is any  prophylactic and/or therapeutic relationship. 
```
Is this the same review? If so, the reference and authors might better be mentioned in the first sentence.

RLordan commented:
> Vitamine D levels as a risk factor and therapeutic is a topic I've also been looking into for a while. Yet, with open questions regarding NLRP3/VDR and VDR/CD147/CypA correlations. Regardless of that, there are three things one might extend investigation now or later on.
> 
> First, the geographic/climatic correlation is biased with skin tone. In Africa, the general population would need more UV exposure than Caucasian ethnicities in Europe and the US. But it would explain the higher fatality rate of the African American community in the US (besides other socio-economic factors of course). The UK and France also have a greater non-Caucasian population due to their historical background.
> 
> Second, if and how Vitamine D deficiency is recognized and treated varies significantly. There might be a correlation among countries with different policies regarding Vitamine D deficiency. The UK might also be affected when compared to Switzerland.
> 
> Third, the role of "Vitamine K" and differentiation of 25- and 1,25-OH. It's possible that it comes down to 1,25-OH and that the findings are biased by Vitamine K levels.
> 
> Lastly, I would ask @rando2 how to proceed with studies with possible conflicts of interest since this study [[10.3390/nu12040988]](https://doi.org/10.3390/nu12040988) was co-funded by a Vitamine D supplier. I think there should be at least a review issue that mentions it for transparency. What do you think?

Thanks for the review @matfax you bring up very interesting points. I will take a look Monday and make some edits :)

matfax commented:
I looked into the UK study and as I suspected, their interpretation seems to be inaccurate. First off, here are two more references regarding the background [[ONS]](https://www.ons.gov.uk/peoplepopulationandcommunity/birthsdeathsandmarriages/deaths/articles/coronavirusrelateddeathsbyethnicgroupenglandandwales/2march2020to10april2020) [[10.3390/nu10040457]](https://doi.org/10.3390/nu10040457). The first reference shows that there is a significantly increased COVID-19 fatality risk for ethnicities with darker skin types in the UK. The second reference created a model that linked Vitamine D deficiency to skin type in the UK. So we can see a correlation there.

Back to the initial UK study. They found a univariable OR=1.37 (p=0.011) to Vitamin D deficiency but then they adjusted it for several factors, including ethnicity, self-reported health rating, age, and several diseases. These factors do all correlate with Vitamine D levels as well. So they might adjust Vitamine D levels for Vitamine D levels. To put it in other terms: Only because aging drives Vitamine D deficiency, is that why Vitamine D deficiency isn't a problem but aging is? They also adjust ethnicities for Vitamine D deficiency and Vitamine D deficiency could account for ca. 10% of the increased fatality risk (their OR of 0.9 is inversed for their interpretation) but with a poor confidence value regardless. Moreover, based on the finding that Vitamine D isn't the primary factor that drives the fatality risk of these ethnicities, they argue as if this unidirectional proof would also neglect that ethnicity could be the primary factor that drives Vitamine D-mediated fatality risk. They are jumping to conclusions. "Our findings do not support a potential link between vitamin D concentrations and risk of COVID-19 infection" - sounds to me like a false claim given their data. Their data supports a "potential link". If the confidences that they provide support a "definite link" can be discussed, of course.

yemarshall commented:
Hello, in case it's harder to see comments on old/closed Pull Requests, mentioning here that I just put a comment on one of your earlier (earliest?) nutraceutical pull request #122 .

RLordan commented:
> Hello, in case it's harder to see comments on old/closed Pull Requests, mentioning here that I just put a comment on one of your earlier (earliest?) nutraceutical pull request #122 .

@yemarshall just replied to it, thanks very much! I look forward to looking through them during the week

RLordan commented:
> Hello, in case it's harder to see comments on old/closed Pull Requests, mentioning here that I just put a comment on one of your earlier (earliest?) nutraceutical pull request #122 .

Hey @yemarshall ,
So, I thought I would answer this point by point to keep all the information together and make it easier for you to follow. I would value your thoughts on the changes and see if you agree with them.
 
1. For the intro part / 4.2.4 Nutraceutical 1st paragraph. Nutraceutical is a new term for me. From the first paragraph it is not fully clear to me what is/are the exact differences (or similarities or overlaps) between the terms: supplement, nutraceutical, and while you're at it distinguishing these from pharmaceutical in a more obvious way. e.g., however, you think it's best to define, maybe with respect to the difference in evidence, fda classification, or any other important things to know

*** So, I have rearranged the definition of nutraceuticals and I have added a lot more information on the background and regulatory information. Let me know if you think it’s too much, but it does provide an explanation to the issues surrounding nutraceuticals and the language used to describe them. 

2. From my reading a few years ago, I remember that words like "treat" or "prevent" were not regarded as good practice for use by Supplements (I'm guessing other related words may not be as well but I'm not sure). At the same time Supplements were allowed to discuss things related to biology somehow but I remember it being very iffy and I remember that sometimes Supplements really pushed the language to imply but not explicitly say what they weren't supposed to say. In case Supplements are related enough to Nutraceuticals, I compiled an up to date list of fda guidelines in case it may be useful (I did notice there was at least one fda warning in the references already, but these may also be useful to double-check language). I put the list at the end of this comment so it doesn't interrupt the flow. I haven't looked through all these myself.

***You are right to a point with regards to supplements, but by definition nutraceuticals should not only supplement the diet, they must also prevent and/or treat a disorder and/or disease to be classified a nutraceutical and not a supplement. The idea around nutraceuticals at the start was that they are naturally sourced nutrients or compounds that induce pharmaceutical like effects to treat or prevent. This was seen as an advantage over pharmaceuticals as nutraceuticals supposedly had less side effects. However, due to poor regulatory framework I'm not sure this has been achieved yet in every case. Here is a good example of a platelet nutraceutical that has been granted a health claim according to EFSA (O'Kennedy, N., Crosbie, L., Song, H. J., Zhang, X., Horgan, G., & Duttaroy, A. K. (2017). A randomised controlled trial comparing a dietary antiplatelet, the water-soluble tomato extract Fruitflow, with 75 mg aspirin in healthy subjects. European Journal of Clinical Nutrition, 71(6), 723-730. doi:10.1038/ejcn.2016.222). This is now a nutraceutical marketed as fruitflow by the company provexis.    

3.In general, I thought the Nutraceutical section was very informative & found just a few places where it would be good to double-check the wording. Here's the first and most obvious one.
The 2nd sentence of first paragraph:
"Nutraceuticals are classified as supplements with health benefits beyond their basic nutritional value designed for the prophylaxis and treatment of disease [160,161]."
I think maybe putting words like "prophylaxis" and "treatment" in the same sentence as the word "supplements" (or "nutraceuticals" depending on the exact definition) might be a very difficult thing unless you double-checked exactly how to do this & then also made extra sure that you are not implying anything you don't want to imply. Even if you make the sentence technically correct, it wouldn't be good to accidently/ambiguously imply something you don't want to. I noticed the rest of the section with the specific examples was in general more conservative in wording or at least it was focused on exact evidence & didn't have these general words. I don't know what to suggest exactly at this stage but I was suggesting you double-check this (and if there's anything I might be qualified to assist with you can let me know).

***I think my response to point two covers this also. However, I see exactly where you are coming from. For instance, vitamin D supplements in normal circumstances are used for those who are deficient or find it difficult to maintain their recommended daily allowance. In this case they are a supplement and all they are intended to do is prevent deficiency. However, in the case of COVID-19, there is an emerging trend that vitamin D and others should be administered in capsule form or infusions due to their potential immunomodulatory capacity. In that capacity, they are for prevention and treatment. I think the extra information provided in the intro section to nutraceuticals highlights the issues surrounding classification etc. If there is anything that needs clarification let me know, or if you think its not clear to the reader. I am happy to go through it.  

4.The last sentence of Zinc supplements Summary subsection has the word "prevent" so it might be necessary to double-check this (& re-word more carefully if needed).

***I think this is ok as it is the intended purpose and the summary suggests maintaining zinc status rather than supplementing. But even so, zinc has been administered with many antiviral drugs in the past so I think this is ok (Trump took zinc with his hydroxychloroquine course!). 

5.The beginning of Vitamin C section has word "prophylactic" (and also check word "therapeutic") so again it might be necessary to double-check this (& re-word more carefully if needed).

***I had to think about this one a bit more. I think you are correct it is more therapeutic than prophylactic. However, taken before the incidence of a flu, the infection severity reduces. So, I have deleted prophylactic for now, but I’m not sure what term to use instead.


6.Straightforward technical correction: The Vitamin C summary subsection last sentence refers to Vitamin C supplementation but I think, based on the preceding subsections, that it was meant to refer to Vitamin C infusion (which may be a different category than supplementation? at least it's a distinction from oral ingestion).

***I have put “supplementation or infusion” there for now as it is only infusion if the patient is ventilated.


Thank you very much, if @rando2 or anyone else wants to chime in please do!

rando2 commented on content/20.treatments.md:
```diff
@@ -322,7 +322,17 @@ Further investigation of HCQ in large, rigorous, multi-center clinical trials is
 Considering the current pandemic, scientists and the medical community are scrambling to repurpose or discover novel host-directed therapies for which nutraceuticals hold some promise.
-Nutraceuticals are classified as supplements with health benefits beyond their basic nutritional value designed for the prophylaxis and treatment of disease [@doi:10.1080/10408390902841529; @doi:10.1038/nrcardio.2016.103].
+Nutraceuticals are classified as supplements with health benefits beyond their basic nutritional value [@doi:10.1080/10408390902841529; @doi:10.1038/nrcardio.2016.103].
+The key difference between a dietary supplement and a nutraceutical is that nutraceuticals should not only supplement the diet, but they must also aid in the prophylaxis and/or treatment of a disorder or disease [@doi:https://doi.org/10.1208/ps050325].
```
```suggestion
The key difference between a dietary supplement and a nutraceutical is that nutraceuticals should not only supplement the diet, but they must also aid in the prophylaxis and/or treatment of a disorder or disease [@doi:10.1208/ps050325].
```

RLordan commented:
> Changes look good to me! I'll wait for a thumbs up from @yemarshall and then merge?

Thanks @rando2 , I am looking into the other comments by matfax before resolving those issues.

rando2 commented:
> Thanks @rando2 , I am looking into the other comments by matfax before resolving those issues.

Ah ok! I can hold off -- just tag me if you need me!

RLordan commented:
> Vitamine D levels as a risk factor and therapeutic is a topic I've also been looking into for a while. Yet, with open questions regarding NLRP3/VDR and VDR/CD147/CypA correlations. Regardless of that, there are three things one might extend investigation now or later on.
> 
> First, the geographic/climatic correlation is biased with skin tone. In Africa, the general population would need more UV exposure than Caucasian ethnicities in Europe and the US. But it would explain the higher fatality rate of the African American community in the US (besides other socio-economic factors of course). The UK and France also have a greater non-Caucasian population due to their historical background.
> 
> Second, if and how Vitamine D deficiency is recognized and treated varies significantly. There might be a correlation among countries with different policies regarding Vitamine D deficiency. The UK might also be affected when compared to Switzerland.
> 
> Third, the role of "Vitamine K" and differentiation of 25- and 1,25-OH. It's possible that it comes down to 1,25-OH and that the findings are biased by Vitamine K levels.
> 
> Lastly, I would ask @rando2 how to proceed with studies with possible conflicts of interest since this study [[10.3390/nu12040988]](https://doi.org/10.3390/nu12040988) was co-funded by a Vitamine D supplier. I think there should be at least a review issue that mentions it for transparency. What do you think?

Hey @matfax  
Thanks for reviewing! 
Point 1: So in terms of vitamin D deficiency, it is a big issue globally and certainly there are several varied reasons for that including a persons demographics, social status, income, lifestyle, sun exposure, skin tone, diet, the list goes on… It is very complex, for that reason it probably deserves a review of its own. 

Point 2. Your second point is most likely very true. As research evolves, we will see more of these studies come to the fore. The one issue is that these vitamin D policies are almost non-existent. There are not many specific national schemes to diagnose and treat vitamin D deficiency. Plenty guidelines advise people to take supplements, promote safe sun exposure, or increase dietary vit D through other mechanisms, but there is very little follow-up to even measure adherence of those guidelines. Vitamin D deficiency is mostly diagnosed on presentation of symptoms or co-diagnosed with other issues (meaning by accessory to another issue or by chance). 

Point 3. Potentially so, likely calcium too. It is often advised that vitamin K be supplemented with vitamin D for calcium absorption and bone health. In the case of covid, the point of administering vitamin D as nutraceutical is not just for maintaining vitamin D levels or treating deficiency, but it is also for its immunomodulatory functions. The latter being of therapeutic value along with restoring baseline or healthier vitamin D levels in patients. 

On your point with conflicts of interest, this particular paper has come up in discussion with @rando2  before when reviewing this section. While there is certainly conflicts of interest in that paper specifically, there is also plenty other papers out there in agreement with them. However, my feeling is that they shouldn’t have pushed such high doses without further evidence, along with a few other assumptions they mention. We still await the results of clinical trials that are underway to discern if vitamin D will be an effective therapeutic. Most of those have end dates between January and May next year! Unfortunately, it is too early to say at this stage whether these papers were right all along or if there suggested doses were a fallacy driven by researcher bias.

RLordan commented:
> I looked into the UK study and as I suspected, their interpretation seems to be inaccurate. First off, here are two more references regarding the background [[ONS]](https://www.ons.gov.uk/peoplepopulationandcommunity/birthsdeathsandmarriages/deaths/articles/coronavirusrelateddeathsbyethnicgroupenglandandwales/2march2020to10april2020) [[10.3390/nu10040457]](https://doi.org/10.3390/nu10040457). The first reference shows that there is a significantly increased COVID-19 fatality risk for ethnicities with darker skin types in the UK. The second reference created a model that linked Vitamine D deficiency to skin type in the UK. So we can see a correlation there.
> 
> Back to the initial UK study. They found a univariable OR=1.37 (p=0.011) to Vitamin D deficiency but then they adjusted it for several factors, including ethnicity, self-reported health rating, age, and several diseases. These factors do all correlate with Vitamine D levels as well. So they might adjust Vitamine D levels for Vitamine D levels (i.e. effect modification). To put it in other terms: Only because aging drives Vitamine D deficiency, is that why Vitamine D deficiency isn't a problem but aging is? They also adjust ethnicities for Vitamine D deficiency and Vitamine D deficiency could account for ca. 11% of the increased fatality risk (their OR of 0.9 is inversed for their interpretation) but with a poor confidence value regardless. Moreover, based on the finding that Vitamine D isn't the primary factor that drives the fatality risk of these ethnicities, they argue as if this unidirectional proof would also eliminate the possibility that ethnicity is the primary factor that drives Vitamine D-mediated fatality risk. They are jumping to conclusions. "Our findings do not support a potential link between vitamin D concentrations and risk of COVID-19 infection" - sounds to me like a false claim given their data. Their data supports a "potential link". If the confidences that they provide support a "definite link" can be discussed, of course.

RLordan commented:
> I looked into the UK study and as I suspected, their interpretation seems to be inaccurate. First off, here are two more references regarding the background [[ONS]](https://www.ons.gov.uk/peoplepopulationandcommunity/birthsdeathsandmarriages/deaths/articles/coronavirusrelateddeathsbyethnicgroupenglandandwales/2march2020to10april2020) [[10.3390/nu10040457]](https://doi.org/10.3390/nu10040457). The first reference shows that there is a significantly increased COVID-19 fatality risk for ethnicities with darker skin types in the UK. The second reference created a model that linked Vitamine D deficiency to skin type in the UK. So we can see a correlation there.
> 
> Back to the initial UK study. They found a univariable OR=1.37 (p=0.011) to Vitamin D deficiency but then they adjusted it for several factors, including ethnicity, self-reported health rating, age, and several diseases. These factors do all correlate with Vitamine D levels as well. So they might adjust Vitamine D levels for Vitamine D levels (i.e. effect modification). To put it in other terms: Only because aging drives Vitamine D deficiency, is that why Vitamine D deficiency isn't a problem but aging is? They also adjust ethnicities for Vitamine D deficiency and Vitamine D deficiency could account for ca. 11% of the increased fatality risk (their OR of 0.9 is inversed for their interpretation) but with a poor confidence value regardless. Moreover, based on the finding that Vitamine D isn't the primary factor that drives the fatality risk of these ethnicities, they argue as if this unidirectional proof would also eliminate the possibility that ethnicity is the primary factor that drives Vitamine D-mediated fatality risk. They are jumping to conclusions. "Our findings do not support a potential link between vitamin D concentrations and risk of COVID-19 infection" - sounds to me like a false claim given their data. Their data supports a "potential link". If the confidences that they provide support a "definite link" can be discussed, of course.

Hey @matfax, I think you made very interesting and valid points there. I think they are trying to be very conservative in their estimations and want to see a clear link between vitamin D and covid, I wouldn’t necessarily think it is an intended false claim as such. They present that vitamin D was associated with COVID-19 infection univariably before adjustments, so clearing a link to begin with. You are probably correct in that there is most likely effect modification. To your point on aging, age is a risk factor for covid in itself due to low-grade inflammation associated with senescence (difficult to measure), not all elderly are vitamin D deficient so I think it is a very complex situation with a multitude of factors at play. Feel free to open an issue on this paper if it is of interest to you. I just know that everyone in the nutrition field right now is mostly being conservative in relation to covid in case they are wrong, be that studies or nutritional guidance. However, the review you pointed out with the conflict of interest certainly is an exception, they wanted to get their ideas across!!!

RLordan commented:
> > Thanks @rando2 , I am looking into the other comments by matfax before resolving those issues.
> 
> @RLordan Ah ok! I can hold off -- just tag me if you need me!

Hey @rando2 I think we are there now unless you feel further review is required. I will continue to update section as studies come to light. Thank you!

agitter commented on content/20.treatments.md:
```diff
@@ -341,8 +351,18 @@ Another potential mechanism by which n-3 PUFA could exert beneficial effects aga
+The effectiveness of n-3 PUFA against infections would be dependent on the dosage, timing, and the specific pathogens responsible [@doi:10.1016/j.jinf.2016.10.001].
+
+One pathological aspect of COVID-19 infection that was late to be recognized is the increased risk of thrombotic complications [@doi:10.1016/j.thromres.2020.04.013]. 
+Considering that there is significant evidence that n-3 fatty acids and other fish oil-derived lipids possess antithrombotic properties and anti-inflammatory properties [@doi:10.1016/j.prostaglandins.2018.09.005;@doi:10.1016/j.blre.2020.100694], they may have therapeutic value against the prothrombotic complications of COVID-19. 
```
```suggestion
Considering that there is significant evidence that n-3 fatty acids and other fish oil-derived lipids possess antithrombotic properties and anti-inflammatory properties [@doi:10.1016/j.prostaglandins.2018.09.005; @doi:10.1016/j.blre.2020.100694], they may have therapeutic value against the prothrombotic complications of COVID-19. 
```

agitter commented on content/20.treatments.md:
```diff
@@ -411,17 +431,29 @@ Vitamin D can be sourced through diet or supplementation, but it is mainly biosy
+Despite these two observational studies potentially linking vitamin D with COVID-19, an examination of the UK Biobank did not support this thesis [@doi:10.1016/j.dsx.2020.04.050]. 
+This analysis examined 25-hydroxyvitamin D concentrations in 348,598 UK Biobank participants, of which 449 were SARS-CoV-2 positive. 
+There is significant interest in the link between vitamin D and COVID-19, hence the multitude of studies published in the literature of varying quality and the clinical trials underway.
+One trial is currently investigating the utility of vitamin D as an immune-modulating agent by monitoring whether administration of vitamin D precipitates an improvement of health status in non-severe symptomatic patients infected with COVID-19 or whether vitamin D prevents patient deterioration [@url: https://clinicaltrials.gov/ct2/show/NCT04334005].
```
```suggestion
One trial is currently investigating the utility of vitamin D as an immune-modulating agent by monitoring whether administration of vitamin D precipitates an improvement of health status in non-severe symptomatic patients infected with COVID-19 or whether vitamin D prevents patient deterioration [@url: https://clinicaltrials.gov/ct2/show/NCT04334005].
```
```suggestion
One trial is currently investigating the utility of vitamin D as an immune-modulating agent by monitoring whether administration of vitamin D precipitates an improvement of health status in non-severe symptomatic patients infected with COVID-19 or whether vitamin D prevents patient deterioration [@url:https://clinicaltrials.gov/ct2/show/NCT04334005].
```
